src/game.ts

```diff
@@ -96,16 +96,29 @@ export default class Game {
     this.map = new Board(
       this,
       this.canvas as HTMLCanvasElement,
-      this.coloniesNumber,
+      this.coloniesNumber
     );
     this.intervalPointer = setInterval(this.play.bind(this), this.speed);
-    this.htmlConnector.$statisticsPanel.innerHTML = "";
+
+    let tableBody = "";
     for (let i: number = 0; i < this.coloniesNumber; i++) {
-      this.htmlConnector.$statisticsPanel.innerHTML +=
-        '<span style="color:' + this.colours[i] + '"></span><br>';
+      tableBody += `<tr class="colony-table__row colony-table__accent" data-team-color="${this.colours[i]}"></tr>`;
     }
+
+    this.htmlConnector.$statisticsPanel.innerHTML = `<table class="colony-table">
```

Comment:
I would rather use `document.createElement` for safety reasons, but it's ok here as there are no server data.